types/snapchat-lens-studio/index.d.ts

```diff
@@ -0,0 +1,4038 @@
+// Type definitions for non-npm package snapchat-lens-studio 3.2
+// Project: https://lensstudio.snapchat.com/api/
+// Definitions by: Olen Davis <https://github.com/OlenDavis>
+//                 Julien Chaumond <https://github.com/julien-c>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// Minimum TypeScript Version: 4.1
+
+// This alone doesn't work when `@types/node` is present:
+declare const global: SnapchatLensStudio.Global;
+
+// // When `@types/node` is present, please add this shim will be needed to establish the typings `global`:
+// declare module NodeJS {
+//     interface Global extends SnapchatLensStudio.Global {}
+// }
+
+/** Returns the time difference in seconds between the current frame and previous frame. */
+declare function getDeltaTime(): number;
+
+/** Returns the time in seconds since the lens was started. */
+declare function getTime(): number;
+
+/** Returns true if the passed in object is null or destroyed. Useful as a safe way to check if a SceneObject or Component has been destroyed. */
+declare function isNull(reference: object): boolean;
+
+/** Prints out a message to the Logger window. */
+declare function print(message: any): void;
+
+/**
+ * Binds scripts to Events and executes them when triggered. Any script can access the ScriptComponent executing them through the variable script. See also: Scripting Overview, Script Events Guide.
+ * ```
+ * // Bind a function to the MouthOpened event
+ * function onMouthOpen(eventData)
+ * {
+ * 	print("mouth was opened");
```

Comment:
<!--
  1 failure:  The root .editorc...
  0 warning: 
  
  
  DangerID: danger-id-Danger;
  File: types/snapchat-lens-studio/index.d.ts;
  Line: 34;
-->
- :no_entry_sign: The root .editorconfig style specifies spaces for whitespace. Please use spaces in new or changed types.